components/src/List/List.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import {
+  space, color, fontSize, fontWeight, lineHeight,
+} from "styled-system";
+import theme from "../theme";
+import ListItem from "./ListItem";
+
+const List = styled.ul`
+  list-style: none;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
+  padding-left: ${theme.space[4]};
+  ${space}
+  ${color}
+  ${fontSize}
+  ${fontWeight}
+  ${lineHeight}
+
+  ${ListItem} {
+    margin-bottom: ${props => (props.compact ? "0" : props.theme.space[2])};
+  }
+`;
+
+List.propTypes = {
+  ...space.propTypes,
+  ...color.propTypes,
+  spacing: PropTypes.bool,
+};
+
+List.defaultProps = {
+  mb: 4,
+  color: "inheretColor",
```

Comment:
inheret -> inherit?

Also, is this different from `currentColor`?

Reply:
changed to currentColor